### PR TITLE
Fix Django CORS error

### PR DESCRIPTION
**Pull request by RezenkovD**

### CODE REVIEW

File: site_dive_into/settings.py [hunk 1]
```diff
@@ -15,7 +15,6 @@
 # Build paths inside the project like this: BASE_DIR / 'subdir'.
 BASE_DIR = Path(__file__).resolve().parent.parent
 
-
```
Comment (qqlexa) [inline site_dive_into/settings.py:18]: Why?

File: site_dive_into/settings.py [hunk 2]
```diff
@@ -39,6 +38,7 @@
     "django_filters",
     "api",
     "rest_framework",
+    'corsheaders',
```
Comment (qqlexa) [inline site_dive_into/settings.py:41]: So... do we need a standard for it? I prefer `'` quotes but `"` quotes already exist in our code-base.

File: site_dive_into/settings.py [hunk 3]
```diff
@@ -49,6 +49,7 @@
     "django.contrib.auth.middleware.AuthenticationMiddleware",
     "django.contrib.messages.middleware.MessageMiddleware",
     "django.middleware.clickjacking.XFrameOptionsMiddleware",
+    'corsheaders.middleware.CorsMiddleware',
```
Comment (qqlexa) [inline site_dive_into/settings.py:52]: The same quotes issue

File: site_dive_into/settings.py [hunk 4]
```diff
@@ -117,7 +115,6 @@
 
 USE_TZ = True
 
-
```
[the remark this answers is not in the document]
Comment (RezenkovD) [reply site_dive_into/settings.py:120]: Formatting Black

File: site_dive_into/settings.py [hunk 5]
```diff
@@ -86,7 +86,6 @@
     }
 }
 
-
```
[the remark this answers is not in the document]
Comment (RezenkovD) [reply site_dive_into/settings.py:89]: Formatting Black

File: site_dive_into/settings.py [hunk 6]
```diff
@@ -71,7 +72,6 @@
 
 WSGI_APPLICATION = "site_dive_into.wsgi.application"
 
-
```
[the remark this answers is not in the document]
Comment (RezenkovD) [reply site_dive_into/settings.py:74]: Formatting Black

File: site_dive_into/settings.py (hunk 1 above)
Comment (RezenkovD) [reply site_dive_into/settings.py:18]: Formatting Black

File: site_dive_into/settings.py (hunk 3 above)
Comment (RezenkovD) [reply site_dive_into/settings.py:52]: I fixed it. Thank you

File: site_dive_into/settings.py (hunk 2 above)
Comment (RezenkovD) [reply site_dive_into/settings.py:41]: I fixed it. Thank you

File: api/serializers.py [hunk 7]
```diff
@@ -1,10 +1,18 @@
 from rest_framework import serializers
+from rest_framework.fields import ReadOnlyField
```
[the remark this answers is not in the document]
Comment (RezenkovD) [reply api/serializers.py:2]: Without it, the code does not work

**Pull request merged**